Owncloud iOs Client/AppDelegate.m

```diff
@@ -194,9 +202,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     }
 
     //Show TouchID dialog if active
-    if([ManageAppSettingsDB isTouchID]) {
-        [[ManageTouchID sharedSingleton] showTouchIDAuth];
-    }
+     [self performSelector:@selector(checkIfIsNecessaryShowTouchId) withObject:nil afterDelay:secondDelay];
```

Comment:
Because I want to execute the method after the delay, for do that I can use the performSelector or Grand Central Dispatch methods. As in OC iOS project there are more extend the performSelector I use that.